fc/parsing/CompactSyntaxParser.py

```diff
@@ -1807,16 +1838,21 @@ def __del__(self):
         sys.setrecursionlimit(self._original_stack_limit)
 
     def _Try(self, callable, *args, **kwargs):
-        """Try calling the given parse command, increasing the stack depth limit if needed."""
+        """
+        Try calling the given parse command, increasing the stack depth limit
```

Comment:
What max line width are you working to? I think other parts of Web Lab code are using 120.

(really need to set up flake8 config for everything!)

Reply:
I've left it at the standard PEP8 width of 79.
It's the standard, but I personally really like that length as well as it means I can fit an editor and a terminal side by side on my laptop screen.

Reply:
(This change was more because of the `"""Starting like this` which I've always found very ugly. But that's a personal preference sorry!)